rclcpp_components/cmake/rclcpp_components_register_node.cmake

```diff
@@ -57,6 +57,9 @@ macro(rclcpp_components_register_node target)
     message(STATUS "Setting executor non-default value ${executor}")
   endif()
 
+  if("${ARGS_PLUGIN}" MATCHES "^[^:]+:[^:]+$")
+    message(FATAL_ERROR "PLUGIN argument '${ARGS_PLUGIN}' appears to be incorrect. (e.g., 'package_name::ClassName').")
+  endif()
```

Comment:
i think we are not supposed to see this diff in this PR because it is already merged with https://github.com/ros2/rclcpp/pull/2878/files